docs/framework/react/guide/not-found-errors.md

```diff
@@ -78,9 +201,19 @@ export const Route = createFileRoute('/_layout/a')({
 })
 ```
 
-### "Global" Not Found Errors
+### Manually targeting the root route
+
+You can also target the root route by passing `route: '__root__'` to the `notFound` function.
```

Comment:
maybe it would be better to keep the `__root__` id as an implementation detail, and add an abstraction?

Reply:
Technically it's exported as `rootRouteId`

Reply:
then at least use this instead of the raw string in the docs?